src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsPage.js

```diff
@@ -126,7 +126,7 @@ function MoneyRequestParticipantsPage({iou, selectedTab, route}) {
                         ref={(el) => (optionsSelectorRef.current = el)}
                         participants={iou.participants}
                         onAddParticipants={IOU.setMoneyRequestParticipants}
-                        navigateToRequest={(option) => navigateToRequestStep(CONST.IOU.MONEY_REQUEST_TYPE.REQUEST, option)}
+                        navigateToRequest={(option) => navigateToRequestStep(iouType.current, option)}
```

Comment:
Will this be different based on who is looking at the iou?

For instance, if I try to send you money will this be a `MONEY_REQUEST_TYPE.REQUEST` for you and a `MONEY_REQUEST_TYPE.SEND` for me?

Reply:
This is part of the flow when a user taps on the send money button on either global create or on a DM. So it solely depends on the option you have selected in the beginning. So if a userA started the flow then it will always be a `Send` request for them and this page doesn't actually come into the picture for userB.

> For instance, if I try to send you money will this be a MONEY_REQUEST_TYPE.REQUEST for you and a MONEY_REQUEST_TYPE.SEND for me?

I think we figure this out by the accountID and managerID, manager seems to be the payer and accountID is the payee.

<img width="1524" alt="Screenshot 2023-10-05 at 11 56 47 PM" src="https://github.com/Expensify/App/assets/35863227/56be0a4f-7464-4dfe-8e92-1eab6dab428d">